app/2.1.x/configuration.md

```diff
@@ -1083,97 +1182,84 @@ block.
-
-See
-http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_prefer_server_ciphers
-
-Default: `on`
+Default: `auto`
 
 ---
 
-#### nginx_http_ssl_session_tickets
+#### nginx_events_worker_connections
 
-Enables or disables session resumption through TLS session tickets. This is has
-no impact when used with TLSv1.3.
+Sets the maximum number of simultaneous connections that can be opened by a
+worker process.
 
-Kong enables this by default for performance reasons, but it has security
-implications: https://github.com/mozilla/server-side-tls/issues/135
+The special and default value of `auto` sets this value to `ulimit -n` with the
+upper bound limited to 16384 as a measure to protect against excess memory use.
 
-See http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_session_tickets
+See http://nginx.org/en/docs/ngx_core_module.html#worker_connections
 
-Default: `on`
+Default: `auto`
 
 ---
 
-#### nginx_http_ssl_session_timeout
+#### nginx_http_client_header_buffer_size
 
-Specifies a time during which a client may reuse the session parameters. See
-the rationale: https://github.com/mozilla/server-side-tls/issues/198
+Sets buffer size for reading the client request headers.
 
-See http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_session_timeout
+See
+http://nginx.org/en/docs/http/ngx_http_core_module.html#client_header_buffer_size
 
-Default: `1d`
+Default: `1k`
 
 ---
 
-#### nginx_upstream_keepalive
-
-Sets the maximum number of idle keepalive connections to upstream servers that
-are preserved in the cache of each worker process. When this number is exceeded,
-the least recently used connections are closed.
+#### nginx_http_large_client_header_buffers
 
-A value of `NONE` will disable this behavior altogether, forcing each upstream
-request to open a new connection.
+Sets the maximum number and size of buffers used for reading large clients
+requests headers.
 
-See http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive
+See
+http://nginx.org/en/docs/http/ngx_http_core_module.html#large_client_header_buffers
 
-Default: `60`
+Default: `4 8k`
 
 ---
 
-#### nginx_upstream_keepalive_requests
+#### nginx_http_client_max_body_size
 
-Sets the maximum number of requests that can be served through one keepalive
-connection.
-
-After the maximum number of requests is made, the connection is closed.
+Defines the maximum request body size allowed by requests proxied by Kong,
+specified in the Content-Length request header. If a request exceeds this limit,
+Kong will respond with a 413 (Request Entity Too Large). Setting this value to 0
+disables checking the request body size.
 
 See
-http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_requests
+http://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size
 
-Default: `100`
+Default: `0`
 
 ---
 
-#### nginx_upstream_keepalive_timeout
+#### nginx_http_client_body_buffer_size
```

Comment:
this gained a `nginx_http_` prefix. Is the old one still valid for backwards compatibility?